### ISSUE TITLE
WebGLRenderer not exported

### ISSUE THREAD

**Issue by joshmarinacci** [ecsyjs/ecsy-three#7]
the initialize() function returns several entities, including camera, scene, and renderer.  You can make other systems use the camera and scene by creating queries with Camera and Scene, but you can't do that for the renderer because it's component type, WebGLRenderer, is not exported from ecsy-three.  Without this there is no way to get a reference to the domElement or other renderer features.

**Comment by Kif11** [comment]
I'm also wondering how can I get renderer from `initialize()`. I need it to get OrbitControls working.

**Comment by dean-careplanner** [comment]
Example pulled from jumpy balls repo. Looks like you need to run one iteration to get the DOM component

```

let world = new World();

// Initialize the default sets of entities and systems
let data = initialize(world);

world.execute(0.016, 0);

let renderer = data.entities.renderer.getComponent(
    WebGLRendererContext
).value
```

**Comment by shawticus** [comment]
FYI this seems to have been fixed in 0.4.0, as initialize returns the WebGLRenderer and associated entity, can probably close

**Comment by robertlong** [comment]
This is fixed now.
